openfl-workspace/xgb_higgs/.workspace

```diff
@@ -0,0 +1 @@
+current_plan_name: default
```

Comment:
Where is this used?

Reply:
thanks, this has been removed. copied over from a template workspace